### PR TITLE
harmonize cli plugins and instructions for different operating systems

### PR DESCRIPTION
**Pull request by matthiasbernhardt**

### CODE REVIEW

**Review by bpetermannS11 — COMMENTED** [review]
mostly ok. But nova and cinder clients are implicitly installed if you install python-openstackclient. If we still want to explicitly install them we should also add python-keystoneclient

File: user/pages/03.Howtos/02.openstack-cli/docs.de.md [hunk 1]
```diff
@@ -140,22 +136,32 @@ openstack --help
...
 
-Es besteht die Möglichkeit Plugins zu installieren. Dafür fügt man in den nachfolgenden Befehl einfach das entsprechende Plugin ein:
+Es besteht die Möglichkeit, Plugins zu installieren. Dafür fügt man in den nachfolgenden Befehl einfach das entsprechende Plugin ein:
 
 ```shell
 pip install python-<PLUGINNAME>client
 ```
 
-Plugins können dann wie folgt installiert werden:
+Benötigte Plugins für den OpenStack (automatisch als Abhängigkeit installiert):
```
Comment (bpetermannS11) [inline user/pages/03.Howtos/02.openstack-cli/docs.de.md:145]:
```suggestion
Benötigte Plugins für den OpenStack (automatisch als Abhängigkeit von python-openstackclient installiert):
```

File: user/pages/03.Howtos/02.openstack-cli/docs.de.md [hunk 2]
```diff
@@ -140,22 +136,32 @@ openstack --help
...
 
-```shell
-pip install python-heatclient
-```
+* keystone - Identity API
+* nova - Compute API
+* cinder - Block Storage Volume API
+* glance - Image API
```
Comment (bpetermannS11) [inline user/pages/03.Howtos/02.openstack-cli/docs.de.md:150]: for me glanceclient was not installed automatically as a dependency, maybe better move it down to "zusätzlich installiert"

File: user/pages/03.Howtos/02.openstack-cli/docs.de.md [hunk 3]
```diff
@@ -140,22 +136,32 @@ openstack --help
...
 * heat - Orchestration API
 * neutron - Network API
+* designate - Domain Name Service API
+* octavia - Load Balancer API
+* barbican - Key Manager API
+
+Die empfohlenen Plugins können dann wie folgt installiert werden:
+
+```shell
+pip install python-heatclient python-neutronclient python-designateclient python-octaviaclient python-barbicanclient
```
Comment (bpetermannS11) [inline user/pages/03.Howtos/02.openstack-cli/docs.de.md:163]: add glance here

File: user/pages/03.Howtos/02.openstack-cli/docs.en.md [hunk 4]
```diff
@@ -56,17 +56,10 @@ Alternatively running a ["virtual environment" with `virtualenv`](#virtualenv) a
...
 ```shell
-sudo -H pip install \
-    python-openstackclient \
-    python-barbicanclient \
-    python-designateclient \
-    python-glanceclient \
-    python-heatclient \
-    python-neutronclient \
-    python-octaviaclient
+sudo -H pip install python-openstackclient python-heatclient python-neutronclient python-designateclient python-octaviaclient python-barbicanclient
```
Comment (bpetermannS11) [inline user/pages/03.Howtos/02.openstack-cli/docs.en.md:62]: add python-glanceclient

File: user/pages/03.Howtos/02.openstack-cli/docs.en.md [hunk 5]
```diff
@@ -88,16 +81,12 @@ C:\Python27\Scripts>easy_install pip
 
 ### OpenStack Client
 
-After the installation of `PIP` finished, we need to install the OpenStack CLI as last step:
+After the installation of `PIP` finished, we need to install the OpenStack CLI client and recommended plugins, to be able to communicate with the corresponding OpenStack APIs:
 
 ```batch
-C:\Python27\Scripts>pip install python-openstackclient python-barbicanclient python-designateclient python-glanceclient python-heatclient python-neutronclient python-octaviaclient
+C:\Python27\Scripts>pip install python-openstackclient python-heatclient python-neutronclient python-designateclient python-octaviaclient python-barbicanclient
```
Comment (bpetermannS11) [inline user/pages/03.Howtos/02.openstack-cli/docs.en.md:87]: add glance

File: user/pages/03.Howtos/02.openstack-cli/docs.en.md [hunk 6]
```diff
@@ -122,10 +111,10 @@ apt install -q -y python-minimal python-pip
 
 If there are dependency errors alternatively running a ["virtual environment" with `virtualenv`](#virtualenv) also works, to have a separate environment for the OpenStack Client.
 
-After the installation of `PIP` finished, we need to install the OpenStack CLI client and required plugins, to be able to communicate with the corresponding OpenStack API:
+After the installation of `PIP` finished, we need to install the OpenStack CLI client and recommended plugins, to be able to communicate with the corresponding OpenStack APIs:
 
 ```shell
-pip install python-openstackclient python-heatclient python-neutronclient python-designateclient
+pip install python-openstackclient python-heatclient python-neutronclient python-designateclient python-octaviaclient python-barbicanclient
```
Comment (bpetermannS11) [inline user/pages/03.Howtos/02.openstack-cli/docs.en.md:117]: add glance

File: user/pages/03.Howtos/02.openstack-cli/docs.en.md [hunk 7]
```diff
@@ -153,17 +142,26 @@ One has the possibility to install plugins. Place the corresponding plugin name
 pip install python-<PLUGINNAME>client
 ```
 
-Installing the heat plugin:
+Required plugins for OpenStack (automatically installed as dependencies):
```
Comment (bpetermannS11) [inline user/pages/03.Howtos/02.openstack-cli/docs.en.md:145]:
```suggestion
Required plugins for OpenStack (automatically installed as dependencies of python-openstackclient):
```

File: user/pages/03.Howtos/02.openstack-cli/docs.en.md [hunk 8]
```diff
@@ -153,17 +142,26 @@ One has the possibility to install plugins. Place the corresponding plugin name
...
-```shell
-pip install python-heatclient
-```
+* keystone - Identity API
+* nova - Compute API
+* cinder - Block Storage Volume API
+* glance - Image API
 
-Required plugins for the SysEleven Stack:
+Recommended plugins for the SysEleven Stacki (explicitly installed):
```
Comment (bpetermannS11) [inline user/pages/03.Howtos/02.openstack-cli/docs.en.md:152]:
```suggestion
Recommended plugins for the SysEleven Stack (explicitly installed):
```

File: user/pages/03.Howtos/02.openstack-cli/docs.en.md [hunk 9]
```diff
@@ -153,17 +142,26 @@ One has the possibility to install plugins. Place the corresponding plugin name
...
-Installing the heat plugin:
+Required plugins for OpenStack (automatically installed as dependencies):
 
-```shell
-pip install python-heatclient
-```
+* keystone - Identity API
+* nova - Compute API
+* cinder - Block Storage Volume API
+* glance - Image API
```
Comment (bpetermannS11) [inline user/pages/03.Howtos/02.openstack-cli/docs.en.md:150]: for me glanceclient was not installed as a dependency

File: user/pages/03.Howtos/02.openstack-cli/docs.en.md [hunk 10]
```diff
@@ -153,17 +142,26 @@ One has the possibility to install plugins. Place the corresponding plugin name
...
 
-Required plugins for the SysEleven Stack:
+Recommended plugins for the SysEleven Stacki (explicitly installed):
 
 * heat - Orchestration API
 * neutron - Network API
-* designate - DNS API
+* designate - Domain Name Service API
+* octavia - Load Balancer API
+* barbican - Key Manager API
```
Comment (bpetermannS11) [inline user/pages/03.Howtos/02.openstack-cli/docs.en.md:158]: add glance

File: user/pages/03.Howtos/02.openstack-cli/docs.en.md [hunk 11]
```diff
@@ -153,17 +142,26 @@ One has the possibility to install plugins. Place the corresponding plugin name
...
+
+Installing the recommended plugins:
+
+```shell
+pip install python-heatclient python-neutronclient python-designateclient python-octaviaclient python-barbicanclient
```
Comment (bpetermannS11) [inline user/pages/03.Howtos/02.openstack-cli/docs.en.md:163]: add glance

**Review by bpetermannS11 — CHANGES_REQUESTED** [review]
There was one typo.
And for me python-glanceclient was not installed as a dependency of python-openstackclient, so maybe move glanceclient to the explicitly-installed plugins

File: user/pages/03.Howtos/02.openstack-cli/docs.de.md (hunk 2 above)
Comment (mbernhardtSys11) [reply user/pages/03.Howtos/02.openstack-cli/docs.de.md:150]: On my system, this was a dependeny (in the Required: line):
```
$ pip show python-openstackclient
Name: python-openstackclient
Version: 4.0.0
Summary: OpenStack Command-line Client
Home-page: https://docs.openstack.org/python-openstackclient/latest/
Author: OpenStack
Author-email: openstack-discuss@lists.openstack.org
License: UNKNOWN
Location: /Users/mbernhardt/.pyenv/versions/3.7.5/lib/python3.7/site-packages
Requires: python-glanceclient, cliff, python-novaclient, six, osc-lib, openstacksdk, oslo.i18n, Babel, oslo.utils, python-keystoneclient, pbr, keystoneauth1, python-cinderclient
Required-by: python-octaviaclient
```

Comment (mbernhardtSys11) [reply user/pages/03.Howtos/02.openstack-cli/docs.de.md]: I wasn't able to clarify this, so I tried to put it differently.

**Review by bpetermannS11 — APPROVED** [review]

**Pull request merged**